src/baseline_bots/bots/smart_order_accepter_bot.py

```diff
@@ -33,6 +34,8 @@
     smart_select_support_proposals,
 )
 
+Aggressiveness = Enum("Aggressiveness", ["aggressive", "moderate", "friendly"])
```

Comment:
This should be converted to use the class-based syntax.

Reply:
done